chainer/function.py

```diff
@@ -7,26 +7,23 @@
 import chainer
 from chainer import configuration
 from chainer import cuda
-from chainer import flag
 from chainer.utils import type_check
 from chainer import variable
 
 
 def no_backprop_mode():
-    """Disable back-propagation for Variable whose volatile is auto.
+    """Disable back-propagation.
 
-    In the default setting a :class:`~chainer.Variable` object whose
-    ``volatile`` attribute is ``'auto'`` behaves like a **non-volatile**
-    variable. That means such a :class:`~chainer.Variable` object builds a
-    computational graph, consumes memory to store the graph, and you can
-    execute back-propagation for it. With this context such a
-    :class:`~chainer.Variable` object behaves like a **volatile** variable.
-    So, you can easily switch training and evaluation.
+    In this context, Chainer does not make a computational graph.
+    :class:`~chainer.Variable` created in this context does not have
+    reference to the :class:`~chainer.Function` which created the variable.
+    So, you cannot execute back-propagation.
```

Comment:
It is better to make this statement more precise. For example, "you cannot compute gradient with `Variable.backward`".